src/routes/comments.rs

```diff
@@ -101,11 +105,15 @@ pub fn create(blog_name: String, slug: String, form: LenientForm<NewCommentForm>
 }
 
 #[post("/~/<blog>/<slug>/comment/<id>/delete")]
-pub fn delete(blog: String, slug: String, id: i32, user: User, conn: DbConn, worker: Worker) -> Result<Redirect, ErrorPage> {
+pub fn delete(blog: String, slug: String, id: i32, user: User, conn: DbConn, worker: Worker, searcher: Searcher) -> Result<Redirect, ErrorPage> {
     if let Ok(comment) = Comment::get(&*conn, id) {
         if comment.author_id == user.id {
             let dest = User::one_by_instance(&*conn)?;
-            let delete_activity = comment.delete(&*conn)?;
+            let delete_activity = comment.build_delete(&*conn)?;
+            Inbox::handle(&Context::build(&conn, &searcher), serde_json::to_value(&delete_activity).map_err(Error::from)?)
```

Comment:
this line seems to be failing on my local instance. I haven't searched why yet

Reply:
If it is exactly on this line, the only thing that can fail is the JSON serialization I think. But that doesn't make sense?? I'll see if I can reproduce.

Reply:
I've added a debug print on the line above and the line under, I get one but not the other (I consider with::<...> and done to be on the same line, they are the same statement)